UPGRADING.md

```diff
@@ -1,3 +1,7 @@
+# Upgrading to 0.16.x
```

Comment:
Since its not a breaking change lets tag this 0.15.1, can you move this text to changelog.md ?

Reply:
updated!